### DISCUSSION TITLE
website is not displayed correctly

### DISCUSSION THREAD

**Discussion by Tealk** [upptime/upptime#932] [Q&A] [answered]
Hi,

I wanted to ask what I have set incorrectly so that the website is not displayed correctly. I wanted to test the tool once.
https://tealk.github.io/upptime/

**Comment by AnandChowdhary** [comment, marked as answer]
You need to make the following changes to https://github.com/Tealk/upptime/blob/master/.upptimerc.yml:

- "repo" should be "upptime", not "/upptime"
- Uncomment the line `# baseUrl: /your-repo-name` and write `baseUrl: /upptime`

**Reply by Tealk** [reply]
Ok I had this before without / which didn't work either and added it because it is written here:
https://upptime.js.org/docs/faq/#my-status-website-is-broken-css-is-not-loading

ah I overlooked the baseurl, sorry
